contracts/truefi2/TrueFiPool.sol

```diff
@@ -0,0 +1,392 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.6.10;
+
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import {SafeMath} from "@openzeppelin/contracts/math/SafeMath.sol";
+
+import {ERC20} from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
```

Comment:
Do we need this ERC20?

Reply:
IERC20 does not have name and symbol functions

Reply:
just use upgradable erc20